scalding-core/src/main/scala/com/twitter/scalding/TypedDelimited.scala

```diff
@@ -47,15 +48,15 @@ trait TypedSeperatedFile extends Serializable {
 /**
  * Typed tab separated values file
  */
-object TypedTsv extends TypedSeperatedFile {
-  val separator = "\t"
+object TypedTsv {
+  def apply[T: TypeDescriptor](path: String*): TypedTextDelimited[T] = TypedText.tsv[T](path: _*)
 }
 
 /**
  * Typed comma separated values file
  */
-object TypedCsv extends TypedSeperatedFile {
```

Comment:
:) too bad we don't have smart spell check for code.

Let's leave it as is for compatibility.

On Friday, July 10, 2015, Sidharth Kapur <notifications@github.com> wrote:

> In scalding-core/src/main/scala/com/twitter/scalding/TypedDelimited.scala
> <https://github.com/twitter/scalding/pull/1356#discussion_r34380731>:
>
> >  }
> >
> >  /**
> >   * Typed comma separated values file
> >   */
> > -object TypedCsv extends TypedSeperatedFile {
>
> top lel, every instance of TypedSeperatedFile in our code base spells the
> word "separated" wrong!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twitter/scalding/pull/1356/files#r34380731>.
>


-- 
Oscar Boykin :: @posco :: http://twitter.com/posco